domain/test/db.async.test.js

```diff
@@ -128,6 +128,26 @@ const coll = 'items';
             assert(returned2[0].type === 'axe');
         });
 
+        it('should be fully to update item', async function () {
```

Comment:
Jeez!! I finally get it now! You've meant "should fully update item"! By the way, if you are unsure what preposition to use, it's probably better to use none.